src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -130,11 +139,30 @@ export const MyVideosView = () => {
                 handleVideoClick(video.id, { minimized: true })
               }}
               onEditVideoClick={() => handleVideoClick(video.id)}
-              onDeleteVideoClick={() => {
-                // TODO: handle delete
-              }}
+              onDeleteVideoClick={() => handleDeleteVideoClick(video.id)}
             />
           ))}
+      <MessageDialog
+        title="Do you want to remove this video from your videos?"
```

Comment:
```suggestion
        title="Do you want to remove this video from your channel?"
```
I think makes more sense

Reply:
will apply Klaudiusz suggestion here https://github.com/Joystream/atlas/pull/544#discussion_r626435019